core/domain/skill_services.py

```diff
@@ -329,3 +329,82 @@ def save_skill_summary(skill_summary):
     )
 
     skill_summary_model.put()
+
+
+def create_skill_mastery(user_id, skill_id, degree_of_mastery):
+    """Creates and stores skill mastery of a user.
+
+    Args:
+        user_id: str. The user id of the logged in user.
+        skill_id: str. The unique id of the skill.
+        degree_of_mastery: float. The language code of the skill.
+    """
+    model_id = user_id + '-' + skill_id
```

Comment:
Ditto here, this would be skill_domain.SkillsMastery.get_model_id(user_id, skill_id).